examples/todo-list/src/models/todo.model.ts

```diff
@@ -41,3 +41,7 @@ export class Todo extends Entity {
     super(data);
   }
 }
+
+export interface TodoRelations {}
```

Comment:
`Todo` already have some relations defined (I think it belongs to TodoList?), can you add navigational properties for these relations to this interface please?